tsconfig.json

```diff
@@ -1,27 +1,29 @@
 {
   "compilerOptions": {
     "baseUrl": "./",
+    "rootDir": ".",
     "paths": {
-      "vision-camera-ocr": ["./src/index"]
+      "@ismaelmoreiraa/vision-camera-ocr": ["./src/index"]
```

Comment:
There is an issue in your patch here !
but as this repo is not maintained ...
I think I will use this PR.

Thank you for sharing it !